central/activecomponent/datastore/internal/store/postgres/gen.go

```diff
@@ -1,3 +1,5 @@
 package postgres
 
 //go:generate pg-table-bindings-wrapper --type=storage.ActiveComponent --search-category ACTIVE_COMPONENT --references storage.Deployment,storage.ImageComponent
```

Comment:
This one has a key change I think between postgres and rocksdb versions that we'll need to look out for

Reply:
Yes, they are converted though imagetoos map